### DISCUSSION TITLE
Cast `tableoid` to `regclass`

### DISCUSSION THREAD

**Discussion by jaulz** [PostgREST/postgrest#3322] [Q&A] [answered]
Is it possible to select the `tableoid` and get the textual representation? I have some inherited tables and I would like to select the parent table and see to which table the records actually belong. I can select the system column `tableoid` but that just returns the internal id. Also, I tried to create a computed column but it throws an error (`ERROR:  column "tableoid" not found in data type entities
LINE 3:   SELECT $1.tableoid::regclass::text;`).

**Comment by wolfgangwalther** [comment, marked as answer]
> I can select the system column tableoid but that just returns the internal id.

Did you try PostgREST's casting syntax on that?

https://postgrest.org/en/v12/references/api/tables_views.html#casting-columns
